README.md

```diff
@@ -54,6 +54,12 @@ any of the following keys:
 Determines if the token secret for the user should be stored in a cookie
 or in `req.session`. Defaults to `false`.
 
+This implements the [Double Submit Cookie Pattern](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#double-submit-cookie) and is
```

Comment:
Sorry, I guess re-reading this part also needs some clarity. It says "This implements the double submit cookie" here, but what does "This" refer to? If it refers to "This" being when the option is set to true (and thus not to false), maybe clarify or move lower to after the description of setting to true is my thought.